ui-lib/src/main/java/co/electriccoin/zcash/ui/screen/account/AndroidAccount.kt

```diff
@@ -82,9 +89,11 @@ internal fun WrapAccount(
     balanceState: BalanceState,
     goBalances: () -> Unit,
     goSettings: () -> Unit,
-    transactionsUiState: TransactionUiState,
+    isHideBalances: Boolean,
     synchronizer: Synchronizer?,
+    onHideBalances: (Boolean) -> Unit,
```

Comment:
view probably should not pass arguments here and we should have `toggleBalances` or `showOrHideBalances`. This would result in having simple lambda () -> Unit. ViewModel would do the toggling for us

`fun setHideBalances(value: Boolean) {
        setBooleanPreference(StandardPreferenceKeys.IS_HIDE_BALANCES, getBooleanPreferences(...))
    }`

What do you think?

Reply:
Addressed in the following commit.